🌐 Add Persian translation for `docs/fa/docs/tutorial/middleware.md`

mojtabapaso opened the pull request:
+ create directory tutorial for this file (middleware.md)

mojtabapaso committed:
Add Persian translation for docs/fa/docs/tutorial/middleware.md

pre-commit-ci[bot] committed:
🎨 [pre-commit.ci] Auto format from pre-commit.com hooks

tiangolo commented:
📝 Docs preview for commit 8279b30230880aa96e08cdb9a23f1634625b58f8 at: https://648df1b211e1f62b3264fab5--fastapi.netlify.app

eghbalpoorMH commented on docs/fa/docs/tutorial/middleware.md:
```diff
@@ -0,0 +1,59 @@
+
+    در صورت وجود هر گونه وظایف پس زمینه (که در ادامه توضیح داده می‌شوند)، تمام میان‌افزارها *پس از آن* اجرا خواهند شد.
+
+## ساخت یک میان نرم‌افزار
```
I think you want to write "میان‌افزار" instead of "میان نرم‌افزار"

eghbalpoorMH commented on docs/fa/docs/tutorial/middleware.md:
````diff
@@ -0,0 +1,59 @@
+{!../../../docs_src/middleware/tutorial001.py!}
+```
+
+ ## سایر میان افزار
````
Your text is not wrong but I think it's better that write like this "میان‌افزار" instead of "میان افزار"

mojtabapaso committed:
Update middleware.md

There was a mistake in the translation, I fixed it and deleted the translation of the technical word "path operation".

tiangolo commented:
📝 Docs preview for commit 58b4f567533da06d654272c5820cbf8354336f64 at: https://64954415fd7f7944dd88ee0e--fastapi.netlify.app

tiangolo committed:
Merge branch 'master' into persian_translate1

tiangolo commented:
📝 Docs preview for commit c69f05fbc3f047b5c5aaacd776f20619098c3c32 at: https://649a31afda0f491e566c44fb--fastapi.netlify.app

tiangolo committed:
Merge branch 'master' into persian_translate1

tiangolo commented:
📝 Docs preview for commit 7fd2b162fa225165fa747ddf643de1f6e4c3805d at: https://649a36047e9b7623acba4b2e--fastapi.netlify.app

mojtabapaso committed:
Merge branch 'master' into persian_translate1

tiangolo commented:
📝 Docs preview for commit c659714bab26da06eef4075144d76beb864daf6e at: https://649bd573e818863a7429f8b4--fastapi.netlify.app

alejsdev commented:
Thank you for your contribution! @mojtabapaso 🤓 

And thanks to everyone for the review ✌️🎉
